libs/core/src/lib/wizard/wizard.component.ts

```diff
@@ -329,18 +354,24 @@ export class WizardComponent implements AfterViewInit, OnDestroy {
     /** @hidden */
     private _setStackedTop(currentStep: WizardStepComponent): void {
         this.steps.forEach((step, index) => {
-            step.getClassList().remove(STEP_STACKED_TOP_CLASS);
-            if (this.steps.toArray()[index + 1] === currentStep) {
-                if (this.steps.length > 1) {
-                    step.getClassList().add(STEP_STACKED_TOP_CLASS);
+            if (step.stepIndicator) {
```

Comment:
Instead of this `if`, you can use filter at the beginning of array

Reply:
This would be more code and harder to read in my opinion

Reply:
Both are equal imo, let's can keep `if` then